src/runner.ts

```diff
@@ -16,6 +16,22 @@ const idColumn = 'id'
 const nameColumn = 'name'
 const runOnColumn = 'run_on'
 
+const migrateSqlFile = (sqlPath: string): MigrationBuilderActions => {
+  const actions: MigrationBuilderActions = {}
+  const regex = new RegExp('^--+s*downsmigration', 'im')
```

Comment:
You can simplify this too `const regex = /^--+\s*down\smigration/im`
Besides, it would be great if you find a better name than `regex` :)